README.md

```diff
@@ -34,30 +34,6 @@ See the [CHANGELOG](CHANGELOG.md).
 - There few demo jobs in the instance so that a user can launch a job on the agent.
 - Kakfa Manager supported in localhost:9000 to support monitoring of Kafka cluster.
 
-6. Stop the demo: `make stop`
```

Comment:
Maybe you didn't plan to delete it

Reply:
@oleg-nenashev I used `clean` command instead of `stop`. I will restore this.